inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,6 +43,16 @@ def __init__(self, protein_pdb: str, temperature: float = 300):
         self.protein_pdb = files.resolve_path(protein_pdb)
         self.temperature = temperature
 
+    def set_small_files(self):
```

Comment:
Call this `get_default_output_files`, for consistency with the wind tunnel scenario. See #518.